kernel/kernel-api/src/main/java/io/delta/kernel/Operation.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (2020-present) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel;
+
+/**
+ * An operation that can be performed on a Delta table.
+ * <p>
+ * An operation is tracked as the first line in delta logs, and powers {@code DESCRIBE HISTORY} for
```

Comment:
inside commit info, right? worth mentioning that?

Reply:
ok, added.